### ISSUE TITLE
Galacean Spine disappear when at the edge of scene (Due to Bounds cut off)

### ISSUE THREAD

**Issue by YunYouJun** [galacean/engine-spine#84]
## Problem

When we move spine to the edge of scene, the spine disappeared.

```ts
// spineEntity.transform.setPosition(0, -15, 0)
spineEntity.transform.setPosition(27.9, -15, 0)
```

We just need to move a little bit, and the spine will disappear immediately.
But it should still show a little spine.

## Reproduction

https://stackblitz.com/edit/b1ookm?file=index.ts

![image](https://github.com/galacean/engine-spine/assets/25154432/be55a960-4148-4834-95a1-235880a9426c)
![image](https://github.com/galacean/engine-spine/assets/25154432/e3689788-3ef7-4c97-b1db-2cdbacfb8327)

## Reason

This is caused by bounds of spine.

The bounds of spine is a point.

We can update the bounds manually to temporarily fix it.

```ts
const spineAnimation = spineEntity.getComponent(SpineAnimation)
spineAnimation.autoUpdateBounds = true
// or
// spineAnimation.updateBounds()
```

![image](https://github.com/galacean/engine-spine/assets/25154432/565965bb-5bfc-4b5a-b21a-030f1ce20e07)

Now, it works well.

## Expected

Spine, which is not truncated by default, updates bounds as needed.

**Comment by johanzhu** [comment]
For performance considerations, the calculation of the spine bounding box was not enabled by default. 
The new version will consider calculating the bounding box once in the first frame to avoid situations where the bounding box is just a point. 🥹

**Comment by johanzhu** [comment]
Fixed in 4.2 version
